CHANGES.rst

```diff
@@ -30,5 +30,7 @@ Version 0.5.0
 -------------
 
 - Service methods became able to specify its error type. [:issue:`71`]
+- Add `nirum-server` command to run nirum service simply.
```

Comment:
- Backquotes should be double.
- The first letter of “nirum” need to be capitalized.
- “simply” should be followed by “run.”